budget/app.py

```diff
@@ -1,14 +1,176 @@
 import json
 import logging
 
+import boto3
+from botocore.exceptions import ClientError
+import requests
+
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
+# TODO configure this URL?
+SYNAPSE_TEAM_MEMBER_URL = 'https://repo-prod.prod.sagebase.org/repo/v1/teamMembers'
+
+# TODO pull this config from S3?
+USER_BUDGET_BY_TEAM = {
```

Comment:
There are a ton of parameters to creating a budget[1].  I guess the idea here is to create a simple budget?  is the `period` setting correspond to the `TimeUnit `?   Maybe we should give these settings similar key names to what the API uses?

[1] https://docs.aws.amazon.com/aws-cost-management/latest/APIReference/API_budgets_CreateBudget.html

Reply:
@zaro0508 -- this part is in progress, and I'd like you to skip looking at it in this PR, as it is currently being edited heavily for the next PR in the series. Regardless, not all the budget params will be defined in this dictionary, just some that vary by team. The key names do not have to match the API.